### ISSUE TITLE
The latejoin menu during hijack should tell you there is a hijack

### ISSUE THREAD

**Issue by Skarletto** [RMC-14/RMC-14#4715]
new player opening the join menu's gonna be confused, even I was till it clicked, ah, it's hijack

**Comment by deltanedas** [comment]
isnt latejoining disabled on hijack

**Comment by Vermidia** [comment]
yes but the menu just shows no roles available to pick instead of telling you

**Comment by GGlut** [comment]
just needs like some random flavour text like the "Almayer is currently under lockdown and hypersleep wakeup is disabled" otherwise new player dont know whats going on. Any maybe just a note like "Marine latejoin is disabled until next round please oberserve"

**Comment by AverageNotDoingAnythingEnjoyer** [comment]
Ig this is fixed since you cant join at hijack anyways

**Comment by CatAndHats** [comment]
added in https://github.com/RMC-14/RMC-14/pull/10546
